Gemfile

```diff
@@ -1,6 +1,19 @@
+# frozen_string_literal: true
 source 'https://rubygems.org'
 
 group :jekyll_plugins do
   gem 'github-pages'
-  gem 'jekyll-theme-primer'
+  gem 'jekyll-remote-theme'
+  gem 'webrick'
+  gem 'jekyll-seo-tag'
+  gem 'jekyll-remote-theme'
```

Comment:
Looks like you added this twice!

Reply:
Sorry about that! I've updated the dependencies in [75483e2](https://github.com/tekknolagi/isdt/pull/22/commits/75483e20c82fa2c80b5d7ffbfdfc2f49729d3f9a), thanks for pointing this out.